src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -16,6 +16,8 @@
     <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
     <NoSemVer20>true</NoSemVer20>
     <IsPackable>true</IsPackable>
+    <IsShipping>true</IsShipping>
+    <IsShipping Condition=" '$(PreReleaseVersionLabel)' == 'preview' ">false</IsShipping>
```

Comment:
Oh, my slight concern for this option is what might happen if we rebrand for an RC release prior to RTM. Should the site extensions ship in that RC?

Reply:
As long as we remember, this is of course fine.

Reply:
I think it should be fine for an RC